zerver/management/commands/check_redis.py

```diff
@@ -26,10 +26,7 @@ def add_arguments(self, parser: CommandParser) -> None:
     def _check_within_range(self, key: str, count_func: Callable[[], int],
                             trim_func: Optional[Callable[[str, int], None]]=None) -> None:
         user_id = int(key.split(':')[1])
-        try:
-            user = get_user_profile_by_id(user_id)
-        except Exception:
-            user = None
+        user = get_user_profile_by_id(user_id)
         entity = RateLimitedUser(user)
         max_calls = max_api_calls(entity)
```

Comment:
With the now removed handling, this would have just failed here instead.

Given that there is no way for the function to proceed without a valid user object I thought it best to just let it fail.